config.yml

```diff
@@ -1,9 +1,14 @@
 ---
   synapse:
     master_fileview: 'syn20446927'
+    manifest_folder: 'manifests'
     manifest_filename: 'manifests/synapse_storage_manifest.csv'
```

Comment:
I notice manifest folder got added as a config param; and we have manifest filename which still contains the 'manifests' folder in the path. Is this intentional?

Reply:
`manifest_filename` is being used in `schematic.synapse.store` on [L#83](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/synapse/store.py#L83), to explicitly point to the file in the `manifests` folder. The `manifests` folder (new parameter added to the config) is to point to the folder where the generated manifest (after running `$ python examples/manifest_generator.py`) should sit.

So in summary, the two config parameters are unrelated and we want to keep both of them.